PWGHF/TableProducer/treeCreatorDsToKKPi.cxx

```diff
@@ -80,6 +80,7 @@ DECLARE_SOA_COLUMN(AbsCos3PiK, absCos3PiK, float);                           //!
 // Events
 DECLARE_SOA_COLUMN(IsEventReject, isEventReject, int); //! Event rejection flag
 DECLARE_SOA_COLUMN(RunNumber, runNumber, int);         //! Run number
+DECLARE_SOA_COLUMN(Sign, sign, int);                   //! Sign
```

Comment:
```suggestion
DECLARE_SOA_COLUMN(Sign, sign, int8_t);                   //! Sign
```
can you test if this works?

Reply:
Thanks for your suggestion! It works, I have implemented it